src/en/administration/install_ansible.md

```diff
@@ -11,6 +11,9 @@ git clone https://github.com/LemmyNet/lemmy.git
 cd lemmy/ansible/
 cp inventory.example inventory
 nano inventory # enter your server, domain, contact email
+# If deploying to an ARM machine, open the file ansible/VERSION,
+# and add -arm64 to the version
+#
```

Comment:
I guess this is fine as a temprary fix until https://github.com/LemmyNet/lemmy/issues/1366 is implemented.